src/components/button/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './styles.module.css';
+
+const Button = React.memo(props => {
+    const { children, Style, onClick } = props;
```

Comment:
name it style and not Style, no attribute starts with uppercase

Reply:
Done